### PR TITLE
Karma module

### PR DESCRIPTION
**Pull request by LinuxMercedes**

### CODE REVIEW

**Comment by drusepth** [comment]
:ship: it

File: modules/karma.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
+var bot;
+
+var scoreboard = {};
+
+loadKarma = function(cb) {
+  bot.fsGetData('karma', 'karma.json', function(err, res) {
```
Comment (streed) [inline modules/karma.js]: Mongo will automatically memory map this, so use mongo.

Comment (LinuxMercedes) [inline modules/karma.js]: oh ok

Comment (hfukada) [inline modules/karma.js]: i feel like redis is a better option

Comment (LinuxMercedes) [inline modules/karma.js]: I feel like your mother is a better option

Comment (hfukada) [inline modules/karma.js]: you cis-normal trans-denying heteronormative socialist.

Comment (streed) [inline modules/karma.js]: :8ball:

Comment (gmackie) [inline modules/karma.js]: YOLOLOLOL

**Pull request merged**